examples/chat_completion/chat_completion_functions_rest.py

```diff
@@ -0,0 +1,160 @@
+from os import environ
+import json
+import requests
+
+from mona_openai import get_rest_monitor
+
+MONA_API_KEY = environ.get("MONA_API_KEY")
+MONA_SECRET = environ.get("MONA_SECRET")
+MONA_CREDS = {
+    "key": MONA_API_KEY,
+    "secret": MONA_SECRET,
+}
+
+# This is the name of the monitoring class on Mona
+MONITORING_CONTEXT_NAME = "MONITORED_CHAT_COMPLETION_USE_CASE_NAME"
+# Get Mona logger
+mona_logger = get_rest_monitor(
+    "ChatCompletion",
+    MONA_CREDS,
+    MONITORING_CONTEXT_NAME,
+)
+
+# Set up the API endpoint URL and authentication headers
+url = "https://api.openai.com/v1/chat/completions"
+headers = {
```

Comment:
url and headers can (should?) be all caps

also i'd give a more indicative name than "url" - would make the usage more readable